pkg/infra/teams/team3/agent3.go

```diff
@@ -83,6 +85,7 @@ func (a *AgentThree) UpdateInternalState(baseAgent agent.BaseAgent, history *com
 		if cmdline.CmdLineInits.PersistentSanctions {
 			a.activeSanctionMap = sanctions.GlobalSanctionMap
 			a.sanctionHistory = sanctions.GlobalSanctionHistory
+			a.sanctionLength = sanctions.GlobalSanctionLength
```

Comment:
Probably need a reference here to work for global sanctions

Reply:
we can delete this and just use `sanctionHistory`